src/actions/guides/database/models.cr

```diff
@@ -240,6 +252,66 @@ class Guides::Database::Models < GuideAction
 
     > The associations *must* be declared on both ends (the Post and the Tag in this example),
     > otherwise you will get a compile time error
+
+    ## Polymorphic
+
+    [Polymorphism](https://en.wikipedia.org/wiki/Polymorphism_(computer_science)) is describes
+    the concept that objects of different types can be accessed through the same interface.
+
+    This allows us to have a single method to define an association, but that method can return
+    many different types. A bit confusing, but best explained with some code!
+
+    ```crystal
+    class User < BaseModel
+      table do
+        has_many events : Event
```

Comment:
Thoughts on doing a Photo, Video, Comment and commentable? I think that might be more immediately understood. "We want to be able to add comments to multiple types of objects. We can do that with a polymorphic association"